src/managers/MessageManager.js

```diff
@@ -36,16 +36,16 @@ class MessageManager extends CachedManager {
    * `after` are mutually exclusive. All the parameters are optional.
    * @typedef {Object} ChannelLogsQueryOptions
    * @property {number} [limit=50] Number of messages to acquire
-   * @property {Snowflake} [before] ID of a message to get the messages that were posted before it
-   * @property {Snowflake} [after] ID of a message to get the messages that were posted after it
-   * @property {Snowflake} [around] ID of a message to get the messages that were posted around it
+   * @property {Snowflake} [before] THe message's id to get the messages that were posted before it
+   * @property {Snowflake} [after] THe message's id to get the messages that were posted after it
+   * @property {Snowflake} [around] THe message's id to get the messages that were posted around it
```

Comment:
```suggestion
   * @property {Snowflake} [before] The message's id to get the messages that were posted before it
   * @property {Snowflake} [after] The message's id to get the messages that were posted after it
   * @property {Snowflake} [around] The message's id to get the messages that were posted around it
```

Reply:
I think we can document it like this: "Get the messages that were posted around this message id" and similar for the other two.
There are other places in the lib where we have done it like this to avoid the weird sentence structure.